package.json

```
@@ -367,7 +367,10 @@
     "build": "webpack --mode production",
     "vscode:prepublish": "npm run build",
     "watch": "webpack --watch --mode development --env esbuild --info-verbosity verbose",
-    "open-in-browser": "vscode-test-web --extensionDevelopmentPath=. ."
+    "open-in-browser": "vscode-test-web --extensionDevelopmentPath=. .",
+    "lint": "eslint . --ext .ts",
+    "lint-fix": "eslint . --ext .ts --fix",
+    "prettier": "prettier --config .prettierrc 'src/**/*.ts' --write"
```

The current call is duplicating what is already being done in `lint-fix`, this will touch all other files instead
```suggestion
    "prettier": "prettier --write ."
```

---

That will also fix all the mixed tabs you have in the format files (if you also add the `.json` extensions)